### ISSUE TITLE
"Invalid command" message does not check nickserv/help is loaded

### ISSUE THREAD

**Issue by ProgVal** [atheme/atheme#808]
With `nickserv/help` module not loaded:

```
C -> S: PRIVMSG NickServ :HELP
S -> C: :nickserv!nickserv@services.int NOTICE chkNS :Invalid command. Use /msg nickserv help for a command listing.
```

**Comment by aaronmdjones** [comment]
Already fixed in master; atheme/atheme@bba5a6172bfb60153311

**Comment by ProgVal** [comment]
Oops! thanks
